server_admin/topics/authentication/webauthn.adoc

```diff
@@ -0,0 +1,197 @@
+|It tells the WebAuthn API implementation on the browser (https://www.w3.org/TR/webauthn/#webauthn-client[WebAuthn Client]) the preference of how to generate an Attestation Statement. However, {project_name} does not verify Attestation Statement so that only "none" option can be selected. This is optional configuration item and applied to the operation of registering WebAuthn authenticator. If no option is selected, its behavior is the same as selecting "none".
+ The further detailed description can be found in the https://www.w3.org/TR/webauthn/#attestation-convey[WebAuthn Specification].
+
+|Authenticator Attachment
+|It tells the WebAuthn Client an acceptable attachment pattern of a WebAuthn authenticator. This is optional configuration item and applied to the operation of registering WebAuthn authenticator. If no option is selected, the WebAuthn Client does not consider the attachment pattern.
+ The further detailed description can be found in the https://www.w3.org/TR/webauthn/#enumdef-authenticatorattachment[WebAuthn Specification].
+
+|Require Resident Key
+|It tells the WebAuthn authenticator to generate the Public Key Credential as https://www.w3.org/TR/webauthn/#client-side-resident-public-key-credential-source[Client-side-resident Public Key Credential Source]. This is optional configuration item and applied to the operation of registering WebAuthn authenticator. If no option is selected, its behavior is the same as selecting "No".
+ The further detailed description can be found in the https://www.w3.org/TR/webauthn/#dom-authenticatorselectioncriteria-requireresidentkey[WebAuthn Specification].
+
+|User Verification Requirement
+|It tells the WebAuthn authenticator to confirm actually verifying a user. This is optional configuration item and applied to both the operation of registering WebAuthn authenticator and authenticating the user by WebAuthn authenticator. If no option is selected, its behavior is the same as selecting "preferred".
+ The further detailed description can be found in the https://www.w3.org/TR/webauthn/#dom-authenticatorselectioncriteria-userverification[WebAuthn Specification for registering WebAuthn authenticator] and https://www.w3.org/TR/webauthn/#dom-publickeycredentialrequestoptions-userverification[WebAuthn Specification for authenticating the user by WebAuthn authenticator].
+
+|Timeout
+|It specifies the timeout value in seconds for registering WebAuthn authenticator and authenticating the user by WebAuthn authenticator. If set to 0, its behavior depends on the WebAuthn authenticator's implementation. The default value is 0.
+ The further detailed description can be found in the https://www.w3.org/TR/webauthn/#dom-publickeycredentialcreationoptions-timeout[WebAuthn Specification for registering WebAuthn authenticator] and https://www.w3.org/TR/webauthn/#dom-publickeycredentialrequestoptions-timeout[WebAuthn Specification for authenticating the user by WebAuthn authenticator].
```

Comment:
Try: "For more details, see...

Reply:
Thank you. I'll fix this sentence as you suggested.